commands/reshowcase

```diff
@@ -225,11 +226,55 @@ if (isBuild) {
       process.exit(1);
     })
     .then((ctx) => {
-      return ctx.serve({ port: port, servedir: outputPath });
+      return ctx.serve({ servedir: outputPath })
```

Comment:
kind of minor: as esbuild will [pick up a random port](https://esbuild.github.io/api/#serve-return-values) if none is defined, and the esbuild server is created before the proxy one, maybe we could set esbuild port as `port + 1`?

Reply:
To ensure that esbuild doesn't accidentally take useful port?

Reply:
Yeah to make sure that esbuild picks port 3000 and user passes 3000 as well from command line. I think it's very unlikely though if esbuild picks the first available port... feel free to ignore.